### ISSUE TITLE
It would be nice to have an option to specify JAVA_HOME

### ISSUE THREAD

**Issue by fp024** [snehaljha/local-tomcat#18]
Hello.

It would be nice to have an option and relevant feature in the local-tomcat settings in settings.json to specify the JAVA_HOME to use when running Tomcat.

```
{
  "local-tomcat.tomcatInstances": [
    {
      "name": "tomcat-10.x",
      "catalinaHome": "C:\\Tomcat\\10",
      "deploymentPort": 8080,
      "javaHome": "C:\\JDK\\17"  // <-- 💡
    },  
    {
      "name": "tomcat-9.0.x",
      "catalinaHome": "C:\\Tomcat\\9",
      "deploymentPort": 8080,
      "javaHome": "C:\\JDK\\11"  // <-- 💡
    }
  ],
  "local-tomcat.warDir": "target"
}
```

I thought it would be better if I could set it up the same way as above, somehow.

Thanks. Have a nice day. 👍

**Comment by snehaljha** [comment]
Hey, I like the idea. It's something good to have. Will definitely look into this. Thanks for your suggestion.

**Comment by snehaljha** [comment]
It has been implemented in v2.1.0.

**Comment by fp024** [comment]
Hello.

I confirmed that Tomcat runs well with the new "javaHome" setting.

Thank you for your support. 👍

However, "Tomcat Stop" does not work well after upgrading to version 2.1.0.

* https://github.com/snehaljha/local-tomcat/blob/051852041b8d9dcd393bfbd6f685fb121ca0998a/src/util/extension-util.ts#L248

Perhaps JAVA_HOME needs to be set in the environment variable when running Tomcat Stop. Could you please check this?

💡 My Windows OS environment does not have JAVA_HOME set system-wide.

**Comment by snehaljha** [comment]
It's fixed in 2.1.1

**Comment by fp024** [comment]
I checked that 'Tomcat Stop' works fine in version 2.1.1 and that Tomcat terminates properly when VSCode is closed.

Thank you. Have a nice day. 👍
